internal/database/gofermart.go

```diff
@@ -0,0 +1,188 @@
+package database
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/nikolaevs92/praktikum-diploma-project.git/internal/objects"
+)
+
+func (gdb *GofermartDB) GetOrderIfExist(order string) (objects.OrderRow, bool) {
```

Comment:
предпочтительнее было бы так 
```suggestion
func (gdb *GofermartDB) GetOrder(order string) (objects.OrderRow, error) {
```
если ордер не существует, то можно отдавать определенный инстанс ошибки

Reply:
так хотя бы будет понятно выше ордера реально нет или это какая-то ошибка

Reply:
Да, согласен, наследие изначальной идеи функции